data/synapse-data-redis/pom.xml

```diff
@@ -37,7 +43,7 @@
         <dependency>
             <groupId>redis.clients</groupId>
             <artifactId>jedis</artifactId>
-            <version>3.3.0</version>
+            <version>3.8.0</version>
```

Comment:
can we add a sample module with a test scenario of this working as part of the sample examples?

Reply:
Sure, I will put together `sample-service-rest-redis-book` example.

Reply:
great.